pypeit/spectrographs/keck_deimos.py

```diff
@@ -204,10 +206,56 @@ def get_detector_par(self, det, hdu=None):
         detector_dict8.update(dict(
             det=8,
             dataext=8,
-            darkcurr=3.69,
+            darkcurr=3.70,
             gain=np.atleast_1d(1.230),
             ronoise=np.atleast_1d(2.580),
         ))
+
+        if hdu is not None:
+            # raw frame date in mjd
+            date = time.Time(self.get_meta_value(self.get_headarr(hdu), 'mjd'), format='mjd').value
+            # get the measurements files
+            measure_files = np.array(glob.glob(f'{data.get_spectrograph_gain_ronoise_dirpath("keck_deimos")}/*'))
```

Comment:
This would change to:

`measure_files = np.array(glob.glob(os.path.join(data.Paths.spectrographs, "keck_deimos", "*")))`

Reply:
Great thanks!